example/notify_inval_entry.c

```diff
@@ -228,7 +228,18 @@ static void tfs_readdir(fuse_req_t req, fuse_ino_t ino, size_t size,
     }
 }
 
+static void tfs_init(void *userdata, struct fuse_conn_info *conn) {
+    (void) userdata;
+    if(options.only_expire && !(conn->capable & FUSE_CAP_EXPIRE_ONLY)) {
+        fprintf(stderr, "FUSE_CAP_EXPIRE_ONLY not supported by kernel\n");
+        exit(1);
+    }
+    if(options.only_expire)
+        conn->want |= FUSE_CAP_EXPIRE_ONLY;
+}
```

Comment:
What is this change about? This is not used in fuse_lowlevel.c `do_init`. See below my comment about kernel side - notifying kernel side that daemon side supports the extended struct would make sense, but then I really do not see why this could not be done in general in `do_init`

Reply:
Sorry, I just adapted this from another example, and didn't get an error.

Happy to fix it, if you have a suggestion.

Reply:
I removed it, as the kernel doesn't actually support this.  Thanks!

Reply:
I still want to exit the filesystem, when the kernel doesn't support what the user asked for.  So I adapted the mechanism from `printcap.c`.